server/app/services/settings/SettingsManifest.java

```diff
@@ -56,6 +56,11 @@ public Optional<String> getWhitelabelCivicEntityFullName(RequestHeader request)
     return getString("WHITELABEL_CIVIC_ENTITY_FULL_NAME", request);
   }
 
+  /** The URL to the civic entity's production CiviForm site. */
+  public Optional<String> getCivicEntityProductionUrl(RequestHeader request) {
+    return getString("CIVIC_ENTITY_PRODUCTION_URL", request);
```

Comment:
(feel free to ignore) would a more clear name be something like `STAGING_SITE_WARNING_BANNER_LINK_TO_PROD_URL`